blocks/ticket/ticket.ts

```diff
@@ -355,7 +355,19 @@ export class Ticket {
             const uploadedFiles = JSON.parse(
               cbk.getVariable(attachmentVar.variable) ?? "[]"
             );
-            for (const uploadedFile of uploadedFiles) {
+            // filter out pdf version of docx file (not display in files tab)
+            const filteredFiles = uploadedFiles.filter(
+              (item: any, _: any, allFiles: any) =>
+                !(
+                  item.fileName.endsWith(".pdf") &&
+                  allFiles.some(
+                    (otherFile: any) =>
+                      otherFile.documentId === item.documentId &&
```

Comment:
suggest another line of comment like this:
```
// when docx file is uploaded, a pdf version is also generated and returned. Both will have same documentId, so we filter out the pdf version here
```

Reply:
better than i could have said